app/src/main/java/de/westnordost/streetcomplete/quests/baby_changing_table/AddBabyChangingTable.java

```diff
@@ -0,0 +1,53 @@
+package de.westnordost.streetcomplete.quests.baby_changing_table;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddBabyChangingTable extends SimpleOverpassQuestType
+{
+	@Inject public AddBabyChangingTable(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways with (((amenity ~ restaurant|pub|cafe|ice_cream|fuel or shop=mall or building=retail) and toilets=yes) or amenity=toilets) and !diaper";
```

Comment:
Originally the opening hours quests allowed for non-named amenities. But it turned out that it is too confusing because it was not clear which shop exactly was meant. Sometimes there is a high density of amenities and oftentimes they are not at the exact correct location. GPS inaccuracy additionally adds to this.

There is a quest that asks for the name of a place, but it has been disabled for now for the same reason. (But sooner or later will be reenabled again.)